src/containters/Home/components/HeroView/index.scss

```diff
@@ -73,10 +73,13 @@
             }
 
             .assistant {
-                border-left: 1px solid #e7e7dc38;
                 height: 75px;
                 display: flex;
-                
+                &::before{
```

Comment:
![Screen Shot 2021-06-11 at 1 38 37 PM](https://user-images.githubusercontent.com/11296684/121680981-83450b00-caba-11eb-8327-64c96c5a141a.png)
![Screen Shot 2021-06-11 at 1 38 49 PM](https://user-images.githubusercontent.com/11296684/121680994-893aec00-caba-11eb-986e-7f81d8c7716b.png)
![Screen Shot 2021-06-11 at 1 39 01 PM](https://user-images.githubusercontent.com/11296684/121681003-8a6c1900-caba-11eb-8d6f-83046104e9d0.png)
![Screen Shot 2021-06-11 at 1 39 10 PM](https://user-images.githubusercontent.com/11296684/121681009-8b9d4600-caba-11eb-849b-fc7026f5da25.png)

Reply:
@AyahElsayed  Please review the screen shots from design VS from app UI